Week-2/HTML-ADVANCED/RESTUARANT-REVIEWS/addReview.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <style>
+        .inputs{
+         margin: 10px 0;
+     }
+
+    </style>
+
+</head>
+
+<body>
+<header>
+    <h1>Restaurant Reviews</h1>
+    <nav>
+        <a href="./index.html">Home</a>
+        <a href="#" class="active">Add Review</a>
+        <a href="./about.html">About</a>
+    </nav>
+</header>
+<main>
+    <h2>Add Review</h2>
+    <form method="post" enctype="multipart/form-data">
+        <div class="inputs">
+            <label>Name?</label>
+            <input type="text" />
+        </div>
+        <div class="inputs">
+            <label>Restaurant?</label>
+            <select >
```

Comment:
`<select name="restaurant">
`

The `select` element should have a name attribute to ensure the form data is submitted correctly. Without a name, the value of this form field will not be sent when the form is submitted.